CE04OSSM/CE04OSSM_D00017_ingest.csv

```diff
@@ -0,0 +1,24 @@
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/syslog/*.syslog.log,CE04OSSM-RID26-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.adcpt_acfgm.dcl.pd0.adcpt_acfgm_dcl_pd0_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/adcpt/*.adcpt.log,CE04OSSM-RID26-01-ADCPTC000,telemetered,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/velpt2/*.velpt*.log,CE04OSSM-RID26-04-VELPTA000,telemetered,Available,
+mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/phsen/*.phsen.log,CE04OSSM-RID26-06-PHSEND000,telemetered,Available,
+mi.dataset.driver.suna.suna_dcl_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/nutnr/*.nutnr.log,CE04OSSM-RID26-07-NUTNRB000,telemetered,Available,
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl26/spkir/*.spkir.log,CE04OSSM-RID26-08-SPKIRB000,telemetered,Available,
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl27/syslog/*.syslog.log,CE04OSSM-RID27-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl27/optaa*/*.optaa.log,CE04OSSM-RID27-01-OPTAAD000,telemetered,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl27/flort/*.flort.log,CE04OSSM-RID27-02-FLORTD000,telemetered,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl27/ctdbp/*.ctdbp.log,CE04OSSM-RID27-03-CTDBPC000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl27/dosta/*.dosta.log,CE04OSSM-RID27-04-DOSTAD000,telemetered,Available,
+# mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/irid2shore/cpm_status*.txt,CE04OSSM-SBC11-00-CPMENG000,telemetered,Pending,Incorrectly specified driver and dataset
+# mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl11/syslog/*.syslog.log,CE04OSSM-SBD11-00-DCLENG000,telemetered,Pending,Incorrectly specified driver and dataset
+# mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CE04OSSM/D00017/cg_data/dcl11/mopak/*.mopak.log,CE04OSSM-SBD11-01-MOPAK0000,telemetered,Not Expected,Not telemetering over Iridium
```

Comment:
@cwingard mopak is commented out here, but not for CE02.  I assume you have a reason, but just wanted to double check one was supposed to be commented out, but not the other?

Reply:
We only get MOPAK data from moorings with cellular modems, so it is commented out for 04, 07 and 09.